test/test_assign/files/example-correct/student/example.ipynb

```diff
@@ -315,6 +315,69 @@
     "grader.check(\"q8\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": false,
+    "editable": false
+   },
+   "source": [
+    "**Question 9:** Testing `otter_include` tag for markdown cells within a solution. Write a function `add`.\n",
+    "\n",
+    "_Points:_ 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "_Type your answer here, replacing this text._"
```

Comment:
This cell appearing is a bug right?

Reply:
I think I need to update these lines to check for the tag? https://github.com/ucbds-infra/otter-grader/blob/e31415bc1c730b6704852c34756078633a225a0d/otter/assign/notebook_transformer.py#L293-L294

Reply:
Thanks for catching that. Yes you should just need to add `and question.manual` onto the condition in that if statement.